src/main/rx/rx_spi.h

```diff
@@ -28,6 +28,7 @@ typedef enum {
     RX_SPI_NRF24_CX10A,
     RX_SPI_NRF24_H8_3D,
     RX_SPI_NRF24_INAV,
+	RX_SPI_NRF24_KN,
```

Comment:
Indentation. Use 4 spaces instead of a tab.

Reply:
Yes, my goal is to connect the NRF24 module through soft spi to an OMNIBUS target and then use the v977 transmitter. I'm working on a custom target to add rxspi and my code.
Sorry for the indentation